src/DGtal/dec/DiscreteExteriorCalculusFactory.h

```diff
@@ -143,6 +160,24 @@ class DiscreteExteriorCalculusFactory
     void
     accumulateAllLowerIncidentCells(const KSpace& kspace, const typename CellsAccum::key_type& cell, CellsAccum& cells_accum);
 
+    /**
+     * Insert and count recursively all lower incident cells into cells accumulator, starting from cell embedding the measure of such cell.
+     * Internal use only.
+     * @tparam KSpace Khalimsky space type.
+     * @tparam CellsAccum cells accumulator type, should be similar to std::map<KSpace::Cell, int> or std::map<KSpace::SCell, int>.
+     * Counts are stored as values, cells are stored as keys.
+     * @param kspace Khalimsky space instance.
+     * @param cell starting cell.
+     * @param cells_accum cells accumulator in which lower incident cells get inserted and counted.
+     * @param local_accum local cells accumulator for measure embedding
+     * @param cell_to_measure a map from cell to its measure
+     * @param measure the current measure to be embedded
+     */
+    template <typename KSpace, typename CellsAccum, typename MeasureAccum>
+    static
+    void
+    accumulateAllLowerIncidentCells(const KSpace& kspace, const typename CellsAccum::key_type& cell, CellsAccum& cells_accum, CellsAccum& local_accum, MeasureAccum& cell_to_measure, const double measure);
```

Comment:
these methods weren't already in the helpers ?

Reply:
The automatic construction of cell measures using a normal estimator was not provided by the factory (although you can do it by hand).